image/contrib/cirrus/runner.sh

```diff
@@ -35,8 +33,11 @@ _run_setup() {
         git checkout FETCH_HEAD
     fi
 
-    msg "Replacing upstream skopeo $SKOPEO_CI_BRANCH branch $project_module module"
-    go mod edit -replace ${project_module}=$GOSRC
+    for subdir in storage image common; do
+        project_module=$(sed -n 's/^module //p' < "$GOSRC/$subdir/go.mod")
```

Comment:
This is not the proper way to parse the information, but I can’t quickly find a clean way.

The previous `cd $dir; go list .` doesn’t work in `common` which has no code in the root of the directory hierarchy.

`cd $dir; go list -m` looks like it _should_ work, but in all three directories it lists all of `…/common, …/image/v5, …/storage`, always in that order. That’s… sort of the data we want, but I don’t _understand_ why it is listing three items, so I’m not comfortable on relying on it.

The `sed` does not handle comments and other syntax, but at least it’s likely to fail if something unexpected happens.

Reply:
> … Ultimately just hard-coding 3 go mod edit lines might be simplest.
 
yeah agree